From: Andreas Noever <andreas.noever@gmail.com>
Date: Sat, 21 Jun 2014 12:15:44 +0200
Subject: [28/31] thunderbolt: select CRC32 in Kconfig
Origin: https://git.kernel.org/linus/f34323b64ac6bb475e158f93c08ee677ecffe8a3

We use __crc32c_le in ctl.c. So make sure that the dependency is there.

Reported-by: kbuild test robot <fengguang.wu@intel.com>
Signed-off-by: Andreas Noever <andreas.noever@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/thunderbolt/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/thunderbolt/Kconfig b/drivers/thunderbolt/Kconfig
index 5aab79b..c121acc 100644
--- a/drivers/thunderbolt/Kconfig
+++ b/drivers/thunderbolt/Kconfig
@@ -1,6 +1,7 @@
 menuconfig THUNDERBOLT
 	tristate "Thunderbolt support for Apple devices"
 	depends on PCI
+	select CRC32
 	help
 	  Cactus Ridge Thunderbolt Controller driver
 	  This driver is required if you want to hotplug Thunderbolt devices on
